src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -51,7 +51,7 @@ class TabulatorSession {
   TabulatorSession(String configPath) {
     this.configPath = configPath;
     // current date-time formatted as a string used for creating unique output files names
-    timestampString = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss").format(new Date());
+    timestampString = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss-SSS").format(new Date());
```

Comment:
Yes -- since Convert tests now test both `Convert to CDF` and `Convert to CSV`, both tests can finish in the same second.

Reply:
Is splitting them into separate tests an option instead, or no? Or maybe sleeping the test threads for 1 second? I just worry about a potential duplicate filename / flaky tests.

Reply:
Running them in the same millisecond is essentially impossible since they're run sequentially, so it won't cause flakiness -- while it's trivial to split into separate tests, bundling them felt nice. Don't feel strongly here either way. Let me know if you prefer them split!

Reply:
I'm coming at it from the opposite end, thinking that it's testing two independent behaviors in the same test (i.e. testing that RCTab converts to CDF properly, and testing that RCTab converts to CSV properly), which makes me lean towards splitting them up. It's entirely possible that I'm misunderstanding this whole situation though, as I haven't been following this whole issue super closely.

Also... what if you ran them on a super badass quantum gaming rig? :trollface:

Reply:
:D 

Fair! I was coming from the "let's avoid duplicate test configs" perspective. Will make the change!

Reply:
I can avoid duplicates by using the same directory for both tests. `output` gets cleared between tests, so there won't be a conflict anymore.